packages/web-components/src/components/toggle-tip/toggletip.mdx

```diff
@@ -1,55 +1,83 @@
-import { ArgTypes, Markdown, Meta } from '@storybook/blocks';
-import { cdnJs, cdnCss } from '../../globals/internal/storybook-cdn';
-import * as ToggleTip from './toggletip.stories';
+import { ArgTypes, Canvas, Markdown, Meta } from '@storybook/blocks';
+import { cdnJs } from '../../globals/internal/storybook-cdn';
+import * as ToggleTipStories from './toggletip.stories';
 
-<Meta of={ToggleTip} />
+<Meta of={ToggleTipStories} />
 
 # Toggletip
 
-> 💡 Check our
-> [Stackblitz](https://stackblitz.com/github/carbon-design-system/carbon/tree/main/packages/web-components/examples/components/toggletip)
-> example implementation.
+[Source code](https://github.com/carbon-design-system/carbon/tree/main/packages/web-components/src/components/toogle-tip)
+&nbsp;|&nbsp;
+[Usage guidelines](https://www.carbondesignsystem.com/components/toggletip/usage)
 
-[![Edit carbon-web-components](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/github/carbon-design-system/carbon/tree/main/packages/web-components/examples/components/toggletip)
+## Table of Contents
 
-The Toggletip component provides an accessible way to render interactive content
-within a popover. If you do not have any interactive content inside of your
-Toggletip, consider using Tooltip instead.
+- [Overview](#overview)
+  - [Toggletips vs Tooltips](#toggletips-vs-tooltips)
+- [Component API](#component-api)
+- [Feedback](#feedback)
 
-## Getting started
+## Overview
 
-Here's a quick example to get you started.
+The `cds-toggletip` component provides an accessible way to render interactive
+content within a popover. If you do not have any interactive content inside of
+your `cds-toggletip`, consider using `cds-tooltip` instead. To use this
+component, you can provide the label by passing it in as a child, include
+actions by using `slot="action"` in a `cds-button`, and to include the content
+you can use the `slot="body-text"`
 
-### JS (via import)
+<Canvas of={ToggleTipStories.Default} />
```

Comment:
This story (`ToggleTipStories.Default`) seems to be bringing in 2 examples. I think we can ditch the second one (has `using open prop` in the label).